### PR TITLE
osa - add prometheus-sidecar

### PR DESCRIPTION
**Pull request by mjudeikis**
change build steps for osa misc repo prometheus sidecar

### CODE REVIEW

File: ci-operator/config/openshift/azure-misc/master.yaml [hunk 1]
```diff
@@ -4,10 +4,10 @@ base_images:
     name: alpine
     namespace: azure
     tag: latest
-binary_build_commands: cd ./src/go; make build-purge
+binary_build_commands: cd make build-all
```
Comment (jim-minter) [inline ci-operator/config/openshift/azure-misc/master.yaml]: ick @mjudeikis this line isn't right :(

Comment (mjudeikis) [reply ci-operator/config/openshift/azure-misc/master.yaml]: ack

Comment (jim-minter) [inline ci-operator/config/openshift/azure-misc/master.yaml]: @mjudeikis need to be using centos image here now

**Pull request merged**